/**/marker_publisher:
  ros__parameters:
    update_period: 0.01
    mesh_pkg: riptide_meshes
    mesh_directory: meshes
    marker_topic: mapping_markers
    robot: talos
    markers:
      marker0:
        mesh: gate
        frame: gate_frame
        scale: 1.0
        pose: [0.0, -1.5, 0.0, 0.0, 0.0, 0.0]
      marker1:
        mesh: earth
        frame: earth_glyph_frame
        scale: 1.0
        pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
      marker2:
        mesh: buoy
        frame: buoy_frame
        scale: 1.0
        pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
      marker3:
        mesh: torpedo
        frame: torpedo_frame
        scale: 1.0
        pose: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
